AndroidProducts.mk

```diff
@@ -16,3 +16,8 @@
 
 PRODUCT_MAKEFILES := \
     $(LOCAL_DIR)/lineage_perry.mk
```

Comment:
This should just be lineage.mk and the makefile renamed to match. Otherwise it is not picked up by lunch

Reply:
Oh yes, I think it was introduced in pie for the aosp way